vespa/test_integration_docker.py

```diff
@@ -1088,6 +1092,20 @@ def tearDown(self) -> None:
         self.vespa_docker.container.stop()
         self.vespa_docker.container.remove()
 
+class TestQaDockerDeploymentImage(TestDockerCommon):
+    def setUp(self) -> None:
+        self.app_package = create_qa_application_package()
+        self.disk_folder = os.path.join(os.getenv("WORK_DIR"), "sample_application")
+
+    def test_deploy(self):
```

Comment:
Instead of creating a new class we can move this `test_deploy` method to the class `TestQaDockerDeployment` that already exists and call it `test_deploy_image`

Reply:
agreed, that will simplify